actioncable/test/javascript/src/unit/subscription_test.js

```diff
@@ -0,0 +1,58 @@
+import consumerTest from "../test_helpers/consumer_test_helper"
+
+const {module} = QUnit
+
+module("ActionCable.Subscription", function() {
+  consumerTest("#initialized callback", ({server, consumer, assert, done}) =>
+    consumer.subscriptions.create("chat", {
+      initialized() {
+        assert.ok(true)
+        done()
+      }
+    }
+    )
```

Comment:
These lines should be combined: `})`. Few more like that in this file too.

Reply:
Updated in https://github.com/rmacklin/rails/commit/c6eaeb060868b6df9f19fd1153f139b20a69dc37